mlflow/server/js/src/common/sdk/UnityCatalogService.js

```diff
@@ -0,0 +1,20 @@
+import { getJson } from '../utils/FetchUtils';
```

Comment:
Do we need this file?

Reply:
We also need to update the conpybara config file to exclude this file.

Reply:
Wondering if it's possible to detect unimported files.

Reply:
Totally unnecessary - deleted :)